Doc/library/sqlite3.rst

```diff
@@ -1279,45 +1284,21 @@ Row objects
    .. versionchanged:: 3.5
       Added support of slicing.
 
-Let's assume we initialize a table as in the example given above::
-
-   con = sqlite3.connect(":memory:")
-   cur = con.cursor()
-   cur.execute('''create table stocks
-   (date text, trans text, symbol text,
-    qty real, price real)''')
-   cur.execute("""insert into stocks
-               values ('2006-01-05','BUY','RHAT',100,35.14)""")
-   con.commit()
-   cur.close()
```

Comment:
This was useful in case I wanted to play around in the interpreter.  Maybe you could have a sample db that you initialize one and then reuse throughout the docs?  It could be the Monty Python movies one from the tutorial, or perhaps another one defined in one of the howto or in a specific section before the reference.

Another solution is to use for the setup the collapsible block of code we discussed about, even though that's not supported by default by Sphinx.

Reply:
> Maybe you could have a sample db that you initialize one and then reuse throughout the docs?

I've though about that, yes. It will require a lot of changes, though.

Reply:
OTOH, it is pretty easy to set up a sample table, we might as well just do that.